eth-contracts/contracts/ServiceProviderFactory.sol

```diff
@@ -477,8 +533,12 @@ contract ServiceProviderFactory is InitializableV2 {
         _updateServiceProviderBoundStatus(_serviceProvider);
     }
 
-    /// @notice Update service provider cut
-    /// SPs will interact with this value as a percent, value translation done client side
+    /**
+     * @notice Update service provider cut of claims
```

Comment:
* Update service provider cut as % of delegate claim, divided by the deployerCutBase. A value of 5 dictates a 5% cut,  with ( 5 / 100 ) * delegateReward going to an SP from each delegator each round.